easybuild/easyconfigs/g/GLPK/GLPK-5.0-GCCcore-12.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLPK'
+version = '5.0'
+
+homepage = 'https://www.gnu.org/software/glpk/'
+description = """The GLPK (GNU Linear Programming Kit) package is intended for
+ solving large-scale linear programming (LP),
+ mixed integer programming (MIP), and other related problems.
+ It is a set of routines written in ANSI C
+ and organized in the form of a callable library."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4a1013eebb50f728fc601bdd833b0b2870333c3b3e5a816eeba921d95bec6f15']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+dependencies = [
+    ('GMP', '6.2.1'),
+]
+
+configopts = '--with-gmp'
+
+
+sanity_check_paths = {
+    'files': ['bin/glpsol', 'include/%(namelower)s.h', 'lib/libglpk.so', 'lib/libglpk.a'],
+    'dirs': [],
+}
```

please keep using `SHLIB_EXT` constant here, and let's also add a sanity check command:

```suggestion
sanity_check_paths = {
    'files': ['bin/glpsol', 'include/%(namelower)s.h', 'lib/libglpk.a', 'lib/libglpk.%s' % SHLIB_EXT],
    'dirs': [],
}

sanity_check_commands = ["glpsol --help"]
```